charts/grafana/values.yaml

```
@@ -959,7 +959,7 @@ sidecar:
     # -- The Docker registry
```

I'm having a limited scope over here, however I believe shouldn't this comment be [RedHat](https://quay.io/repository/kiwigrid/k8s-sidecar?tab=tags) registry instead of Docker ?

---

@jkroepke what are your thoughts ?

---

As I know, quay.io does not have such low rate limits as docker have

---

@jkroepke, Thanks for the context and I am in sync on using quay.io instead of docker, however the comment is kind of misleading since docker registry isn't being used here, therefore my suggestion is to just change the comment to:
```suggestion
    # -- The Red Hat registry
```

---

That makes sense! Thanks for explaining!